dotcom-rendering/src/web/components/StickyBottomBanner/ReaderRevenueBanner.tsx

```diff
@@ -130,6 +136,10 @@ const buildPayload = async ({
 			browserId: (await hasCmpConsentForBrowserId())
 				? browserId || undefined
 				: undefined,
+			// eslint-disable-next-line -- because i do what i like
+			// @ts-ignore
```

Comment:
These are temporary, and I will remove once https://github.com/guardian/support-dotcom-components/pull/723 is merged and the package has been bumped.